src/wallet/wallet.cpp

```diff
@@ -46,6 +46,8 @@ bool fPayAtLeastCustomFee = true;
 
 const char * DEFAULT_WALLET_DAT = "wallet.dat";
 
+NotesHelperIndex* NotesHelperIndex::instance = NULL;
```

Comment:
```suggestion
NotesIndex* NotesIndex::instance = NULL;
```

Reply:
Consider making this belong to a CWallet instance rather than being a global singleton.

Reply:
Done at https://github.com/zcash/zcash/pull/4399/commits/d215e9f3b9b9005468e4bd32e0d3959c7be043f9